src/main/java/sirius/web/http/Response.java

```diff
@@ -835,6 +835,15 @@ protected void installChunkedWriteHandler() {
         }
     }
 
+    protected void removedChunkedWriteHandler(ChannelFuture writeFuture) {
+        writeFuture.addListener(ignored -> {
+            if (ctx.channel().pipeline().get(ChunkedWriteHandler.class) != null && ctx.channel().isOpen()) {
```

Comment:
should we really check if the channel is open? If the client closes the channel will close but the chunkwriter wont be removed?

Reply:
die pipeline + channel + everything will be discarded anyway...